src/transformState.js

```diff
@@ -5,7 +5,23 @@
  * @param {Object[]} actions
  */
 function transformState(state, actions) {
-  // write code here
+  for (const action in actions) {
```

Comment:
```suggestion
  for (const action of actions) {
```
why do you need `for in` for iterating array?

Reply:
@Bohdan-Maksymiuk  done